src/intro_movie_scene_3.c

```diff
@@ -0,0 +1,771 @@
+#include "global.h"
+#include "intro_movie_internal.h"
+#include "system.h"
+#include "gf_gfx_loader.h"
+#include "unk_0200ACF0.h"
+#include "unk_0200B150.h"
+#include "unk_0200FA24.h"
+#include "gf_3d_render.h"
+#include "unk_020215A0.h"
+#include "unk_02022588.h"
+#include "demo/opening/gs_opening.naix"
+#include "unk_02026E30.h"
+
+void IntroMovie_Scene3_VBlankCB(void *pVoid);
```

Comment:
just realized- why aren't these functions defined as static when they're (seemingly) only used here?

Reply:
also pVoid for the file

Reply:
> https://github.com/pret/pokeheartgold/pull/300#discussion_r1527555938

because i didn't get around to it, and it's super tedious anyway